src/extension/tsconfigHelper.ts

```diff
@@ -23,24 +24,27 @@ export class TsConfigHelper {
         return fileSystem.exists(tsConfigPath)
             .then(function(exists: boolean): Q.Promise<void> {
                 if (!exists) {
-                    return fileSystem.writeFile(tsConfigPath, "{}");
+                    return fileSystem.writeFile(tsConfigPath, "{ compilerOptions: { allowJs: true } }");
```

Comment:
@MSLaguana There is already a consider to tsConfig not exists. Do you mean this?